packages/react-core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -199,7 +199,7 @@ export const DatePicker: React.FunctionComponent<DatePickerProps> = ({
         </div>
       </Popover>
       {helperText && <div className={styles.datePickerHelperText}>{helperText}</div>}
-      {errorText && <div className={css(styles.datePickerHelperText, styles.modifiers.error)}>{errorText}</div>}
+      {errorText.trim() && <div className={css(styles.datePickerHelperText, styles.modifiers.error)}>{errorText}</div>}
```

Comment:
errorText can not be null or undefined?

Reply:
OK i looked at where it's coming from, state defaults it to empty string so this is fine